<launch>
  <arg name="oscar_port" default="/dev/ttyACM0" />
  <!-- <arg name="config_filepath" default="$(find oscar_can_joy)/config/$(arg joy_config).config.yaml" /> -->

  <node pkg="oscar_vehicle_driver" name="oscar_vehicle_driver" type="driver.py" output="screen">
    <param name="oscar/port" value="$(arg oscar_port)" />
    <!-- <rosparam command="load" file="$(arg config_filepath)" /> -->
  </node>
</launch>
